src/pages/OfferNew/Checkout/CheckoutContent.tsx

```diff
@@ -103,7 +111,24 @@ export const CheckoutContent: React.FC<Props> = ({
           </div>
         </Excerpt>
 
-        <UserDetailsForm email={email} onEmailChange={onEmailChange} />
+        <UserDetailsForm
+          email={email}
+          onEmailChange={onEmailChange}
+          ssn={firstQuote.ssn ?? ''}
+          onSsnChange={(ssn) => {
+            setFakeLoading(true)
+            setReallyLoading(true)
+            window.setTimeout(() => setFakeLoading(false), 1000)
+            // TODO we somehow need to compare the birth date to the ssn to check so they match. In case they don't, we should warn (as they might get a different price)
```

Comment:
So we use the birthdate to calculate price now right?

Reply:
Yes, apparently norwegians are really careful with giving out their fødelsenummer, so we're pushing that step as late as possible